pages/02_picture.py

```diff
@@ -16,13 +25,18 @@ def png_upload():
     # jpg, jpeg, pngを受け入れるアップローダーを作成してください
     # まず、アップロードされたファイルをtemp folderに格納し、そのデータをエンコードしにいくコードを考えてみると分かりやすいかもです。
     # ただ、できれば無駄なフォルダを作りたくないので、直接エンコードできればなお良しです。
-    return
+    picture = st.file_uploader("画像を選択", type=["png", "jpeg", "jpg"])
+    if picture is None:
+        return None
+    b64_pct = encode_image(picture)
+    return b64_pct
```

Comment:
- if節はこっちの方がシンプルですかね。
```suggestion
    if picture:
        return encode_image(picture)
    else:
        return None
```